src/RedaxtorSeo.js

```diff
@@ -275,4 +275,7 @@ export default class RedaxtorSeo extends Component {
  */
 RedaxtorSeo.__renderType = "BEFORE";
 RedaxtorSeo.__editLabel = i18n.__floatingEditLabel;
-RedaxtorSeo.__name = i18n.__checkboxName;
\ No newline at end of file
+RedaxtorSeo.__name = i18n.__checkboxName;
+RedaxtorSeo.applyEditor = function(node, data){
+
```

Comment:
Fit it. But in fact there is nothing to apply here. All what rendered it is a changing of class name fr active / inactive editor